evm_arithmetization/src/cpu/kernel/interpreter.rs

```diff
@@ -57,6 +57,8 @@ pub(crate) struct Interpreter<F: Field> {
     /// Holds the value of the clock: the clock counts the number of operations
     /// in the execution.
     pub(crate) clock: usize,
+    /// TODO
```

Comment:
Why TODO?

Reply:
I think David was relying on you to write a proper doc for this :)

Reply:
David was indeed relying on you! :)

Reply:
Ah I see, thanks! :) Would 
```suggestion
    /// Maximal number of CPU cycles in one segment execution.
```
be good?